### DISCUSSION TITLE
Update On Collaboration (RUMBus)

### DISCUSSION THREAD

**Discussion by Yahid1** [uprm-inso4116-2024-2025-s1/semester-project-trolley-tracker-app#4] [General]
Yesterday I was in a reunion with the Lead Dev of RUMBus and founder of the NonProfit Org Evolving Space  (Cristian Melendez). He showed me all the features that where completed by his Team, he explained the code, all the technologies that he has and demonstrated the app. His approach is divided into two different applications. One of the applications is directed towards users that will ride the Trolleys the other is directed towards administration as a sort of portal that can be used to update routes, stops, drivers and other administrative data. 

His team has for the most part already completed the application where the client in this case are the riders of the Trolleys. He mentioned needing help on the Administrative app which he needs some features that need to get implemented. He is happy to give a demonstration on how his application works and wants to remind everyone that this is part of learning process where you'll encounter many different technologies. 

### TLDR

If we incorporate collaboration then we will most likely only work on features related to Administrative app of RUMBus. This means making the portal user friendly so that the administrator can update routes, stops and etc. 

Example of the app for the Trolley riders: 
![WhatsApp Image 2024-08-30 at 10 28 57_1f71b544](https://github.com/user-attachments/assets/2dfe3923-8bd9-4537-bdce-1d5b1496c429)

**Comment by Yahid1** [comment]
I wanted to add that all backend, servers and technologies are already implemented and paid for by the organization. This includes the GPS devices are installed on the Busses and another device which monitors the "Vitals" of the Bus. 


### They use: (framework)


RUMBus : React Native 

RUMBus Administrative Dashboard : Angular

**Comment by MrL9000** [comment]
hmm, sounds like a nice opportunity. Admittedly maybe our scope might get limited a bit, but I'd be interested on focusing on the development of this part of the project. This would also give us extra time to catch up with what the developers have coded up to this point to make the process of implementing code. This direction for the project has my full support

**Reply by Yahid1** [reply]
Perfect, Thank you for your input!
